Dockerfile

```diff
@@ -0,0 +1,52 @@
+# hadolint global ignore=DL3008
+FROM debian:buster
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+      python \
+      python-pip \
+      python-dev \
+      python-setuptools \
+      ca-certificates \
+      gcc \
+      libpq-dev \
+      curl \
+      gunicorn \
+    && rm -rf /var/lib/apt/lists/*
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+# kelseyhightower confd arm64 builds are broken: https://github.com/kelseyhightower/confd/issues/850
+ARG CONFD_VERSION=0.19.1
+RUN CONFD_ARCH=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/) \
+    && curl -SL "https://github.com/abtreece/confd/releases/download/v${CONFD_VERSION}/confd-v${CONFD_VERSION}-linux-${CONFD_ARCH}.tar.gz" | tar -xz -C /usr/local/bin/ \
+    && /usr/local/bin/confd -version
+
+RUN groupadd --system --gid 510 osqa && useradd --system --gid osqa --uid 510 osqa
+
+WORKDIR /usr/src/app
+RUN chown osqa:osqa /usr/src/app
```

Comment:
Why `/usr/src` here? It's likely doubly wrong because `/usr` is for the OS and package manager and `/usr/src` if it's anything is historically the OS source directory...

Reply:
Moved it to `/srv/app` the location is fairly arbitrary. The container itself is disposable.

Reply:
I've also now set ARGS (build time only environment variables) which can be overridden if desired to set preferred container UID/GID.